### PR TITLE
docs: update bio for devversion

### PR DESCRIPTION
**Pull request by devversion**
This PR should land as my 1000th commit on the `main` branch (aside from the other projects/branches)

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -307,9 +307,9 @@
   "devversion": {
     "name": "Paul Gschwendtner",
     "picture": "devversion.jpg",
-    "twitter": "DevVersion",
-    "website": "https://github.com/DevVersion",
-    "bio": "Paul is a 17-year-old developer living in Germany. While he attends school, Paul works as a core team member on Angular Material. Paul focuses on tooling and building components for Angular.",
+    "twitter": "devversion",
+    "website": "https://github.com/devversion",
+    "bio": "Paul is a Software Engineer at Google. He is part of the Angular team since 2016.",
```
Comment (alan-agius4) [inline aio/content/marketing/contributors.json:312]: Finally.... I though you were going to stay a 17-year-old developer for eternity :)

Comment (devversion) [reply aio/content/marketing/contributors.json:312]: Hahah 🤣

**Review by AndrewKushnir — APPROVED** [review]
🎉

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit e93708b864e8e479c0c78bda9ff59045513ee209.

**Pull request closed without merging**